src/components/StatueMenuContainer.js

```diff
@@ -5,9 +5,11 @@ import style from "./StatueMenuContainer.module.css";
 export class StatueMenuContainer extends Component {
   render() {
     return (
-      <div className={style.container}>
-        <h1>Statues</h1>
-        <StatueMenu />
+      <div className="col-lg-3 col-md-6 col-sm-12">
```

Comment:
This one should be `col-lg-3 col-md-12`, at the moment it only goes halfway across at medium when the other is fully across underneath